README.md

```diff
@@ -1,12 +1,12 @@
 # optimaze-viewer-example
 
-This is an example application for the [`optimaze-viewer`](https://github.com/rapal/optimaze-viewer) library. 
-It is written in TypeScript and uses Webpack for bundling.
+This is an example application for the [`optimaze-viewer`](https://github.com/rapal/optimaze-viewer) library. It is written in TypeScript and uses Webpack for bundling. It implements the OAuth 2 authorization code flow for authenticating API requests.
 
 Note: The API which this application uses is not yet published but will be published later in 2017.
 
 ## Usage
 
 1. Run `yarn` to install dependencies
-2. Run `yarn start` to start dev server
-3. Alternatively, run `yarn build` and open `index.html`
\ No newline at end of file
+2. Run `yarn start` to run webpack in watch mode
+    * Or run `yarn build` to create a production build
+3. Open `index.html`
```

Comment:
Also add demo user to Optimaze Portal (demo-user@demo.com or something similar) + document it here?